cmd/boulder-va/main.go

```diff
@@ -39,18 +39,18 @@ func main() {
 		go cmd.ProfileCmd("VA", stats)
 
 		pc := &va.PortConfig{
-			SimpleHTTPPort:  80,
-			SimpleHTTPSPort: 443,
-			DVSNIPort:       443,
+			HTTPPort:  80,
+			HTTPSPort: 443,
```

Comment:
I'm not going to mess with SimpleHTTP in this PR.  It's dead tech.  We can pull the HTTPSPort when we kill SimpleHTTP.